tests/flow/test_prev_rdb_decode.py

```diff
@@ -33,3 +33,69 @@ def test_v6_decode(self):
         self.env.assertIn("Index Scan", plan)
         results = redis_graph.query("MATCH (n:L1 {val:1}) RETURN n")
         self.env.assertEqual(results.result_set, [[node0]])
+    
+    def test_v7_decode(self):
+        graph_name = "v7_rdb_restore"
+        # docker run -p 6379:6379 -it redislabs/redisgraph:2.2.8
+        # dump created with the following query (v6 supported property value: integer, double, boolean, string, null, array)
+        #  graph.query g "CREATE (:L1 {val:1, strval: 'str', numval: 5.5, nullval: NULL, boolval: true, array: [1,2,3]})-[:E{val:2}]->(:L2{val:3})"
```

Comment:
Please create additional indexes, ones that actually index data and ones that doesn't e.g.
`CREATE INDEX ON :L1(none_existsing)`

apply this to all decoders